### DISCUSSION TITLE
using another model

### DISCUSSION THREAD

**Discussion by thanhtantran** [imartinez/privateGPT#1517] [Q&A]
i have setup and run smooth the default privateGPT with model [Mistral-7B-Instruct-v0.1-GGUF](https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF)

However, i want to change to another models to test the perfomance, how I can change it?

**Comment by mastnacek** [comment]
in privateGPT/settings.yaml  after change right model name, run "poetry run python scripts/setup"
local:
  prompt_style: "mistral"
  llm_hf_repo_id: TheBloke/Mistral-7B-Instruct-v0.2-GGUF
  llm_hf_model_file: mistral-7b-instruct-v0.2.Q4_K_M.gguf
  embedding_hf_model_name: BAAI/bge-small-en-v1.5

**Reply by WudgeJalk** [reply]
@mastnacek I'm not sure to understand, this is a step we did in the installation process. However it doesn't help changing the model to another one. More over in privateGPT's manual it is mentionned that we are allegedly able to switch between "profiles" ( "A typical use case of profile is to easily switch between LLM and embeddings. To be a bit more precise, you can change the language (to French, Spanish, Italian, English, etc) by simply changing the profile you've selected; no code changes required!") but I have no idea how to do that (nor just how to create a profile) it was quite hard to go trough the installation process. I don't want to ruin everything :/

**Reply by silverbuggy** [reply]
check out this reply.  It gives step by step instructions on how to change the model: https://github.com/imartinez/privateGPT/discussions/1152

**Reply by MiggerX2** [reply]
THANK YOU!!

**Comment by dbzoo** [comment]
See also the answer https://github.com/imartinez/privateGPT/discussions/225
